src/utils/components/CPUMemory/CPUMemory.tsx

```diff
@@ -14,20 +14,21 @@ import { Skeleton } from '@patternfly/react-core';
 import { isRunning } from '@virtualmachines/utils';
 
 type CPUMemoryProps = {
+  fetchVMI?: boolean;
   vm: V1VirtualMachine;
 };
 
-const CPUMemory: FC<CPUMemoryProps> = ({ vm }) => {
+const CPUMemory: FC<CPUMemoryProps> = ({ fetchVMI = true, vm }) => {
   const { t } = useKubevirtTranslation();
   const itMatcher: V1InstancetypeMatcher = vm?.spec?.instancetype;
   const { instanceType, instanceTypeLoaded, instanceTypeLoadError } = useInstanceType(itMatcher);
   const isVMRunning = isRunning(vm);
-  const { vmi, vmiLoadError } = useVMI(getName(vm), getNamespace(vm));
+  const { vmi, vmiLoadError } = useVMI(getName(vm), getNamespace(vm), fetchVMI);
 
   if ((isVMRunning && vmiLoadError) || (!isEmpty(itMatcher) && instanceTypeLoadError))
     return <MutedTextSpan text={t('Not available')} />;
 
-  if ((isVMRunning && !vmi) || !vm || !instanceTypeLoaded) return <Skeleton />;
+  if ((isVMRunning && !vmi) || !vm || !instanceTypeLoaded) return <Skeleton width="100px" />;
```

Comment:
why 100px? try to add a class 
![image](https://github.com/kubevirt-ui/kubevirt-plugin/assets/14824964/e3335b7d-16af-4303-a226-69d8afc7b2b3)